dynamic-layers/intel-ese-bsp/recipes-kernel/linux/linux-intel-ese-lts-acrn-sos_5.4.bb

```diff
@@ -5,3 +5,12 @@ SRC_URI_append = "  file://sos_5.4.scc"
 LINUX_VERSION_EXTENSION = "-linux-intel-ese-lts-acrn-sos"
 
 SUMMARY = "Linux Intel ESE Kernel with ACRN enabled (SOS)"
+
+SRC_URI_append = " ${@get_scenario_cfg(d)}"
+
+def get_scenario_cfg(d):
+    if bb.utils.contains('ACRN_BOARD', 'ehl-crb-b', True, False, d):
+        if bb.utils.contains('ACRN_SCENARIO', 'hybrid_rt_fusa', True, False, d):
+            return 'file://sos_5.4_hybrid_rt_fusa.scc'
+    return ''
+
```

Comment:
Please remove white-space character.

Reply:
thank you I removed the white-space at Line#16